src/main/java/com/nike/cerberus/domain/SafeDepositBox.java

```diff
@@ -164,4 +164,46 @@ public void setUserGroupPermissions(Set<UserGroupPermission> userGroupPermission
     public void setIamRolePermissions(Set<IamRolePermission> iamRolePermissions) {
         this.iamRolePermissions = iamRolePermissions;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        SafeDepositBox that = (SafeDepositBox) o;
+
+        if (id != null ? !id.equals(that.id) : that.id != null) return false;
```

Comment:
What do you think of making these statements positive so that they're easier to read?
e.g. `( ! ( id == null ? that.id == null : id.equals(that.id) ) ) return false;`